x-pack/solutions/search/plugins/enterprise_search/server/plugin.ts

```diff
@@ -149,10 +149,12 @@ export class EnterpriseSearchPlugin implements Plugin<void, void, PluginsSetup,
           ui: [],
         },
         read: {
-          disabled: true,
+          app: ['kibana', ...PLUGIN_IDS],
```

Comment:
I wonder if we need to curtail this list for a viewer? specifically wondering about the onboarding experience and showing the global empty state to a viewer who cannot create an index. But that may require more changes than just including a subset of the `PLUGIN_IDS` here.

Reply:
Yup, I think that would be a next step. We have an issue for it already, just never picked it up.